src/parser/parser.py

```diff
@@ -43,12 +43,11 @@ def _load_vdata(self):
                 )
 
     def _load_localizations(self):
-        # Load all localizations data to memory
+        """
+        Merge all 5 localization groups - attributes, gc, heroes, main, and mods -
```

Comment:
me being pointlessly nitpicky but just for the sake of pointing it out;

I requested some examples so that its more easily interpretable, something like "Merge localization groups (i.e. attributes, heroes, mods)" would also do the job done. you wouldn't want to create more work for yourself as *technically* if they added another localization group now this sentence is either inaccurate or misleading

ik, im practically toying with you, but thought it funny enough to point out

Reply:
Yeah that's true, I just based it off the previous bit about merging 5 groups. But I'll make it more general, since the code is general too